wasm-node/pkg/.gitignore

```diff
@@ -0,0 +1 @@
+*
```

Comment:
I think the correct way is to do
```suggestion
*
!/.gitignore
```
I think this works because you somehow already had this `.gitignore` file in git's index. However if you'd try to do
```bash
mkdir foo
echo '*' > foo/.gitignore
# those won't work
git add foo
git add foo/.gitignore
```

Reply:
wasm-pack generates a .gitignore?

Reply:
Yes. `pkg` directory itself is needed for `rust-embed` to compile at all so I've added it to tracking so that `clippy` et al doesn't fail in CI.